### PR TITLE
add lw7 hackathon article

### PR DESCRIPTION
**Pull request by dshukertjr**
## What kind of change does this PR introduce?

Add LW7 hackathon blog article.

### CODE REVIEW

File: apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx [hunk 1]
```diff
@@ -0,0 +1,114 @@
+---
+title: 'Launch Week 7 Supa AI Hackathon'
+description: Build an Open Source Project, Win $1500 and limited edition Supa AI swag packs!
+author: ant_wilson
+image: lw7-hackathon/thumbnail.jpg
+thumb: lw7-hackathon/thumbnail.jpg
```
Comment (dshukertjr) [inline apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx:6]: Waiting on the design team

File: apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+There are 5 chances to win, there will be prizes for:
+
+- Best Overall Project ($1500 in GitHub Sponsorship & Swag kit)
+    - to be paid as 3x$500 GitHub Sponsorships (over 3 months)
+- Most fun/visually pleasing (Swag kit)
+- Most technically impressive (Swag kit)
+- Most useful (Swag kit)
+- Best use case of [pgvector](https://supabase.com/docs/guides/database/extensions/pgvector) (Swag kit)
+
```
Comment (dshukertjr) [inline apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx:45]: Double check to see if we are happy with the categories

File: apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+You can win $1500 in GitHub sponsorships and official limited edition Supabase swags!
+
+For inspiration, check out all the [winners from previous hackathons](https://supabase.com/blog/tags/hackathon).
+
+![Supabase Hackathon AI Generated Image](/images/blog/lw7-hackathon/supabase-hackathon-ai-generated.png)
+
+## Key Facts
+
+- You have 10 days to build a new **Open Source** project using Supabase that utilizes the power of AI in some capacity.
```
Comment (dshukertjr) [inline apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx:26]: Should we mandate to use "AI in some capacity"?

File: apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+- Best Overall Project ($1500 in GitHub Sponsorship & Swag kit)
+    - to be paid as 3x$500 GitHub Sponsorships (over 3 months)
+- Most fun/visually pleasing (Swag kit)
+- Most technically impressive (Swag kit)
+- Most useful (Swag kit)
+- Best use case of [pgvector](https://supabase.com/docs/guides/database/extensions/pgvector) (Swag kit)
+
+There will be a winner and a runner-up prize for each category. Every team member on winning/runner-up teams gets a Supa AI Hackathon swag kit.
+
+[swag kit image here]
```
Comment (dshukertjr) [inline apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx:48]: Waiting on the design team.

File: apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+## Prizes
+
+There are 5 chances to win, there will be prizes for:
+
+- Best overall project (Supabase Keyboards & Swag kit)
+- Most fun/best easter egg (Swag kit)
+- Most technically impressive (Swag kit)
+- Most visually pleasing (Swag kit)
+- Best AI project (Swag kit)
```
Comment (dshukertjr) [inline apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx:46]: Just want to make sure these were the categories that we wanted.

File: apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx (hunk 1 above)
Comment (MildTomato) [reply apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx:6]: can we do all this in notion?

Comment (dshukertjr) [reply apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx:6]: We can. It's in the LW7 board!
https://www.notion.so/supabase/LW7-Iota-search-Horsey-50ad028d176341b882348300666c246a?p=e6ef8462f5a1460ba74e3d563fce6f23&pm=s

**Review by RamiroND — APPROVED** [review]

File: apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx [hunk 6]
```diff
@@ -0,0 +1,107 @@
+---
+title: 'The Supabase AI Hackathon'
+description: Build an Open Source Project over 10 days. 5 prize categories.
+author: tyler_shukert
+image: lw7-hackathon/thumbnail.png
+thumb: lw7-hackathon/thumbnail.png
```
Comment (fsansalvadore) [inline apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx:6]:
```suggestion
thumb: lw7-hackathon/thumbnail.png
launchweek: 7
```

Comment (fsansalvadore) [reply apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx:6]: Can you please just add this `launchweek: 7` to the meta data?
I'll use it to show some additional stuff at the end of the post.

Comment (dshukertjr) [reply apps/www/_blog/2023-04-07-launch-week-7-hackathon.mdx:6]: Yup, sounds great!

**Pull request merged**